pages/program.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="../styles/program-page.css" rel="stylesheet" />
+    <link href="../styles/program.css" rel="stylesheet" />
+    <title>Hack Your Future Program</title>
+  </head>
+  <body>
+    <div class="page container">
+      <!--header-->
+      <header class="header">
+        <div class="header-container">
+          <div class="logo-container">
+            <a href="/">
+              <img src="././img/hyf1.svg" alt="logo of Hack Your Future" />
+            </a>
+          </div>
+        </div>
+        <nav class="header-nav">
+          <a href="url" class="nav-link1">
+            <span>THE PROGRAM</span>
+          </a>
+
+          <a href="url" class="nav-link2">
+            <span>VOLUNTEER</span>
+          </a>
+
+          <a href="url" class="nav-link3">
+            <span>DIGITALRNTS</span>
+          </a>
+
+          <a href="url" class="nav-link4">
+            <span>ABOUT</span>
+          </a>
+          <a href="" class="nav-link5">
+            <span>FAQ</span>
+          </a>
+          <a href="url" class="nav-link6">
+            <span>SUPPORT US</span>
+          </a>
```

Comment:
Is there any reason why you're rewriting the same style over and over again? You can reuse the nav-link class for each \<a\>-tag.  
```suggestion
        <nav class="header-nav">
          <a href="url" class="nav-link">
            <span>THE PROGRAM</span>
          </a>

          <a href="url" class="nav-link">
            <span>VOLUNTEER</span>
          </a>

          <a href="url" class="nav-link">
            <span>DIGITALRNTS</span>
          </a>

          <a href="url" class="nav-link">
            <span>ABOUT</span>
          </a>
          <a href="" class="nav-link">
            <span>FAQ</span>
          </a>
          <a href="url" class="nav-link">
            <span>SUPPORT US</span>
          </a>
```
Now you can remove all the styles. This makes your code way more manageable in the long run. I've added a suggestion in the .css file.